modules/atomic-widgets/module.php

```diff
@@ -74,4 +79,26 @@ private function enqueue_scripts() {
 			true
 		);
 	}
+
+	/**
+	 * Add the CSS for the element.
+	 *
+	 * @param Post $post
+	 * @param Element_Base $element
+	 */
+	private function add_css( Post $post, Element_Base $element ) {
+		if ( ! ( $element instanceof Atomic_Widget_Base ) || Post::class !== get_class( $post ) ) {
+			return;
+		}
+
+		$styles = $element->get_raw_data()['styles'];
+
+		if ( empty( $styles ) ) {
+			return;
+		}
```

Comment:
I'm not sure whether this piece of code should be a part of this PR or be in the next PR

Reply:
So the next PR will remove it.